src/main/clj/lemur/core.clj

```diff
@@ -938,13 +939,17 @@ calls launch              - take action (upload files, start cluster, etc)
      true ;default if body is empty
      ~@body))
 
+(defn aws-credentials
+  []
+  (DefaultAWSCredentialsProviderChain.))
+
 (defn -main
   "Run lemur help."
   [& [command & args]]
   (add-command-spec* context init-command-spec)
   (let [[profiles remaining] (split-with #(.startsWith % ":") args)
-        aws-creds (awscommon/aws-credential-discovery)
-        jobdef-path (first remaining)]
+        jobdef-path (first remaining)
+        aws-creds (awscommon/aws-credentials)]
```

Comment:
That's right, I initially got rid of common.clj altogether, then saw that the credentials were needed in tests, and kept it.

I will remove it now, and use lemur.core/aws-credentials where needed.